### ISSUE TITLE
Multiple todo files

### ISSUE THREAD

**Issue by azrdev** [mNantern/QTodoTxt#144]
Please add the ability to simultaneously open multiple todo.txt files, and move entries between them

**Comment by mimmori** [comment]
+1
This feature would be very nice. One application could be a shared set of activities (via dropbox)

**Comment by funkyfuture** [comment]
that would break the `--quickadd` cli option if i'm not wrong.

**Comment by kmicc** [comment]
A quite nice idea. But if you have more than one todo file opened and create a new task - in which of the opened todo files should that task be saved? How should this use case be handled?

**Comment by mimmori** [comment]
Have you ever use google calendar?
I think a behaviour like that would be perfect.

You can work with single file view and select different one when you need.

**Comment by kmicc** [comment]
No, I never used google calendar. But what maybe could work is an additional dropbox in the task edit dialog, where you can choose the file the task should be saved to.

**Comment by kmicc** [comment]
On the other hand: Opening multiple instances of QTodoTxt with different todo lists is already possible. So what is actually missing is the possibility to copy/move tasks, see also #150.

**Comment by sttrebo** [comment]
as opposed to a second instance/todo file, how about some sort of persistent filter.  for example, i have a number of "private" actions that i don't necessarily want displayed on my PC at work.  could there be a preference setting outside of the normal filtering that would allow a boolean not "private" to be maintained outside of the standard filtering?
